.github/workflows/tests.yml

```diff
@@ -9,19 +9,19 @@ jobs:
     strategy:
       matrix:
         include:
-          - ruby_version: '2.7'
-            gemfile: rails_7_0
+          - ruby_version: '3.3'
+            gemfile: rails_7_1
             upload_coverage: true
-          - ruby_version: '2.5'
+          - ruby_version: '3.2'
+            gemfile: rails_7_1
+          - ruby_version: '3.1'
+            gemfile: rails_7_1
+          - ruby_version: '3.3'
+            gemfile: rails_7_0
+          - ruby_version: '3.3'
             gemfile: rails_6_1
-          - ruby_version: '2.3'
-            gemfile: rails_5_0
-          - ruby_version: '2.3'
-            gemfile: rails_4_2
-            bundler: '1'
     env:
       CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
-      CI: 1
```

Comment:
[`CI` is already a default variable set to `true` in GHA](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables); no need to set this variable.